README.md

```diff
@@ -1,2 +1,30 @@
 # puppet-hashi_stack
-This module contains shared code for various HashiCorp modules
+
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/hashi_stack.svg)](https://forge.puppetlabs.com/puppet/hashi_stack)
+[![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/hashi_stack.svg)](https://forge.puppetlabs.com/puppet/hashi_stack)
+[![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/hashi_stack.svg)](https://forge.puppetlabs.com/puppet/hashi_stack)
+[![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/hashi_stack.svg)](https://forge.puppetlabs.com/puppet/hashi_stack)
+[![puppetmodule.info docs](http://www.puppetmodule.info/images/badge.png)](http://www.puppetmodule.info/m/puppet-hashi_stack)
+[![Apache-2.0 License](https://img.shields.io/github/license/voxpupuli/puppet-hashi_stack.svg)](LICENSE)
```

Comment:
did you pick apache on purpose for this module or just because it's the github default / how much do you like AGPL? (I'm just happy about every new AGPL licensed module but that's my personal opinion and staying with apache is perfectly fine as well)

Reply:
i have no strong feelings for either other than apache being my personal default. more out of convenience/ocd that conviction. Happy to change